settings.gradle

```diff
@@ -68,5 +67,11 @@ include "solr:distribution"
 include "solr:docker"
 include "solr:prometheus-exporter"
 
+// Do not load the new UI module if it is disabled explicitly
+def withNewUiModule = !rootProject.hasProperty('disableNewUiModule') || rootProject.findProperty('disableNewUiModule') != 'true'
```

Comment:
Wouldn't it be easier just to add a default value here?

Reply:
I tried, but it does not allow me to set `rootProject.ext.*`. I believe it is because it is not yet initialized, since we are in `settings.gradle` / the initialization phase.